src/http/StatusCode.cc

```diff
@@ -276,3 +277,25 @@ Http::StatusCodeString(const Http::StatusCode status)
     return "Unassigned";
 }
 
+bool
+Http::IsReforwardableStatus(const StatusCode s)
```

Comment:
For symmetry with the equivalent method semantic checks this function should be a member of the StatusLine class.

This implies fixing the bug where ```answer.peerResponseStatus``` only passes the status code value, dropping the protocol, version and phrase parts of the server status line on custom/unknown codes.

Reply:
> For symmetry with the equivalent method semantic checks this function should be a member of the StatusLine class.

I disagree on both counts:

* There is not enough symmetry here IMO: HttpRequestMethod represents HTTP method, but Http::StatusLine does not represent HTTP status code.

* Http::StatusLine represents HTTP status line (that has a lot more information than just the status code) while the function in question is specific to the status code. If we start making these reforwarding decisions based on other status line parts, _then_ we would move this function into that class. For now, the function is best placed into the more specific/narrow StatusCode scope.